selfdrive/car/hyundai/values.py

```diff
@@ -292,6 +292,19 @@ class Buttons:
       [HYUNDAI_VERSION_REQUEST_MULTI],
       [HYUNDAI_VERSION_RESPONSE],
     ),
+    # CAN-FD queries
+    Request(
+      [HYUNDAI_VERSION_REQUEST_LONG],
+      [HYUNDAI_VERSION_RESPONSE],
+      whitelist_ecus=[Ecu.fwdRadar],
+      bus=4,
+    ),
+    Request(
+      [HYUNDAI_VERSION_REQUEST_LONG],
+      [HYUNDAI_VERSION_RESPONSE],
+      whitelist_ecus=[Ecu.fwdCamera, Ecu.adas],
+      bus=5,
```

Comment:
what happens if you don't have an aux panda? does it just wait for the timeout?

Reply:
I believe so. I was thinking we could maybe have some generic way to determine if we have an aux panda and only query with those requests, if specified in the config. But not sure how complex that would be

Reply:
This adds around 0.2-0.4 seconds of extra delay